.ci/jenkins/build-int-test/Jenkinsfile

```diff
@@ -142,13 +142,20 @@ pipeline {
     }
     post {
         always {
-            echo "Cleaning up Docker images from local host"
-            sh ".ci/scripts/util/cleanup.sh ${DOCKER_TAG}"
-            echo "Removing int test conda environment"
-            sh "/var/lib/jenkins/conda_installs/${RUN_ID}/miniconda3/bin/conda env remove --yes --name int_test_env"
-            sh "/var/lib/jenkins/conda_installs/${RUN_ID}/miniconda3/bin/conda init --reverse"
-            sh "rm -rf /var/lib/jenkins/conda_installs/${RUN_ID}/"
-            deleteDir()
+            script {
+                catchError (buildResult: 'FAILURE', stageResult: 'FAILURE') {
```

Comment:
Nice find, was just looking at the documentation for `catchError` and it looks like we could also include a message:

> message : String (optional)
> A message that will be logged to the console if an error is caught. If the stage result is specified, the message will also be associated with that result and may be shown in visualizations.

Might be worth adding to make it more obvious when either of the cleanup routines fails.

Reply:
Done